src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -156,12 +152,8 @@ public async Task When_Calling_Select_With_Out_Of_Range_Length()
 
 			WindowHelper.WindowContent = textBox;
 			await WindowHelper.WaitForLoaded(textBox);
-
-#if __WASM__ // Wasm is behaving differently than UWP and other platforms. https://github.com/unoplatform/uno/issues/7016
-			Assert.AreEqual(10, textBox.SelectionStart);
-#else
-			Assert.AreEqual(0, textBox.SelectionStart);
-#endif
+			textBox.Focus(FocusState.Programmatic);
+			Assert.AreEqual(textBox.Text.Length, textBox.SelectionStart);
```

Comment:
This behavior is resetting the caret to the end (selection start is now the text length). It should be 0 instead.

Reply:
In UWP when you focus the caret goes to the end, I equalized the Platforms to UWP behavior.